src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -314,12 +316,13 @@ internal string CreateSearchQuery(string[] fields, string SearchTerm)
             var booleanQuery = new BooleanQuery();
             string searchTerm = QueryParser.Escape(SearchTerm);
 
-            if (searchTerm.Contains('.'))
-                searchType = SearchType.ByCategory;
-            else if (searchTerm.Contains(' '))
-                hasEmptySpaces = true;
-            else
-                searchType = SearchType.Normal;
+            if (!IsPackageContext)
```

Comment:
For packages the `.` would not work right?

Reply:
yes

Reply:
Do you mean that `.` will not be searched?